### ISSUE TITLE
interactive potential graph and control panels "dance" when particles move through a certain range

### ISSUE THREAD

**Issue by jbphet** [phetsims/states-of-matter#176]
There is an issue with the current master versions of States of Matter and Atomic Interactions where the interaction potential diagram (3rd screen of SOM, main screen of AI) shimmies around when the particles move through a certain range.  After a lot of detective work, I found that commenting out the HandNode makes the problem go away.  I showed this to @jonathanolson, and he suggested that I log an issue and tag him.

**Comment by jbphet** [comment]
I've added a partial workaround for this that moves the hand to an out-of-the-way position if it is invisible.  This causes the shimmy to stop when the hand is not visible.

**Comment by jonathanolson** [comment]
Is there more information about reproducing, as I don't seem able to reproduce this. Which browsers/OSes and screens (as it seemed screen-specific?) had this problem, and are there any tips to reproduce?

**Comment by jbphet** [comment]
I just did some testing, and saw this on:
+ Chrome 54.0.2540.99 on Win 10
+ Firefox 49.0.2 on Win 10
+ Firefox 42 on Win 7

I did NOT see it on
+ Chrome Canary 56.0.2920.0 on Win 10
+ Safari on OSX
+ Chrome on OSX
+ Edge on Win 10

@jonathanolson - are you going to be on campus for meetings tomorrow?  If so, I could just demo the behavior.

**Comment by jbphet** [comment]
I just tried the following and saw the effect on both:
- Firefox 50 on OSX 10.12.1
- Firefox 50 on Win 7

**Comment by jonathanolson** [comment]
> @jonathanolson - are you going to be on campus for meetings tomorrow? If so, I could just demo the behavior.

Yes, will be on campus today.

**Comment by jbphet** [comment]
@jonathanolson and I looked at this together, and it appears that the jiggling was due to block sizes changing and thus being "refitted".  Setting `preventFit` to `true` for the `AtomicInteractionsScreenView` causes the symptom to disappear.

I've seen other performance issues in the past with this screen that I would bet are related to this as well.

For the record, the nodes that were pushing out the boundary of the layer were the force arrows, since they grow quite large when the atoms are moved to where they overlap.

I'm unassigning this issue and will have it verified in the next dev or rc release.
